### PR TITLE
DROOLS-6056 : GDST to XLS conversion should wrap String values in a list associated with 'in' with quotes

### PR DESCRIPTION
**Pull request by Rikkola**
**JIRA**:

[DROOLS-6056](https://issues.redhat.com/browse/DROOLS-6056)

[war](https://www.dropbox.com/s/v11qqv337j25ydk/business-central-7.53.0-SNAPSHOT-wildfly19.war?dl=0)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by Rikkola** [comment]
Jenkins retest this

**Comment by Rikkola** [comment]
Failing kie-server tests. Not sure how the plain PR build picks them up, but looks like it did.
Jenkins retest this

**Comment by Rikkola** [comment]
Jenkins retest this

**Comment by Rikkola** [comment]
Jenkins retest this

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/util/CommaSeparatedListValueConverter.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.guided.dtable.backend.server.conversion.util;
+
+import org.apache.commons.lang.text.StrBuilder;
```
Comment (Rikkola) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/util/CommaSeparatedListValueConverter.java:18]: This is wrong. The builds are actually picking something up :)

**Review by jstastny-cz — APPROVED** [review]
LGTM.

**Comment by Rikkola** [comment]
```
sample/src/test/java/org/kie/server/springboot/samples/utils/KeycloakContainer.java:[24,33] package com.github.dockerjava.api does not exist
[2021-04-14T09:36:26.420Z] [ERROR] /home/jenkins/workspace/KIE/master/pullrequest/drools-wb-master.pr/bc/kiegroup_droolsjbpm_integration/kie-spring-boot/kie-spring-boot-samples/keycloak-kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/utils/KeycloakContainer.java:[25,43] package com.github.dockerjava.api.exception does not exist

```
Jenkins retest this

**Comment by Rikkola** [comment]
`You have triggered an abuse detection mechanism. Please wait a few minutes before you try again.`

Jenkins retest this

**Comment by Rikkola** [comment]
Haha, and the 7th build actually gets far enough to find an actual error. Going to fix this. 

```
org.drools.workbench.screens.guided.dtable.backend.server.conversion.XLSBuilderAttributesTest.content
org.drools.workbench.screens.guided.dtable.backend.server.conversion.XLSBuilderCarsInCityTest.content
org.drools.workbench.screens.guided.dtable.backend.server.conversion.XLSBuilderVirusStateTest.content
```

**Comment by Rikkola** [comment]
Jenkins retest this

**Comment by Rikkola** [comment]
@kiegroup/gatekeepers Can I get a merge please.

**Pull request merged**